spec/requests/users_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  context 'GET index' do
```

Comment:
Spec file should include tests that ensure the correct view template is rendered. Example : `expect(response).to render_template(:index)` in your case. Kindly test for the same.